target/linux/ramips/patches-5.4/0300-mtd-rawnand-add-driver-support-for-MT7621-nand-flash.patch

```diff
@@ -1155,9 +1155,11 @@ Signed-off-by: Weijie Gao <weijie.gao@mediatek.com>
 +	uint32_t i, j;
 +	u8 *oobptr;
 +
-+	for (i = 0; i < mtd->writesize; i++)
-+		if (buf[i] != 0xff)
-+			return 0;
++	if (buf) {
++		for (i = 0; i < mtd->writesize; i++)
++			if (buf[i] != 0xff)
++				return 0;
++	}
```

Comment:
you've changed the  number of lines without changing this part:
```
--- /dev/null
+++ b/drivers/mtd/nand/raw/mt7621_nand.c
@@ -0,0 +1,1348 @@
```
which means this patch now becomes invalid.
You should change the 1348 to 1350 as two lines are added.

Reply:
Sorry, I never worked with raw patches before. Thanks